README.md

```diff
@@ -1 +1,18 @@
 # Badgerloop Testing Framework
+
+## Setup
+This repository is designed for and validated on a Raspberry Pi 4 running Ubuntu 22.04 Jammy Jellyfish. The Pi is expected to have the following packages:
+- python3 (Should be included by default on Ubuntu 22.04)
+- python3-venv (sudo apt update && sudo apt install python3-venv)
+- pigpiod (sudo apt update && sudo apt install pigpiod)
+
+Additionally, you will need a testConfig file to indicate what mocks are needed. 
+
+## Running Tests
+1. Run the setup script to create the python environment and fetch all required packages (./setup.sh </folder/to/create/in>)
+2. Activate the python environment (source </folder/you/specified/above>)
```

Comment:
Ok with the argument change. Currently the script is deactivating the venv at the end. I also think step 2 should be separated out since for manual use, this gives context on how to run tests later on without rerunning the entire setup script.

Reply:
What if we don't deactivate the venv at the end of the script then? It seems kind of redundant to deactivate the environment and then reactivate it. Also, by activating the environment using setup.sh each time, we're ensuring that the environment being used is up-to-date. We could add a printout at the end of `setup.sh` to say something like:
```
Currently running in the virtual environment at ${venv_path}. This environment is set up for BTF usage.

If any Python packages need to be installed or updated, exit the virtual environment, update
requirements.txt accordingly, and rerun setup.sh.

You can exit this virtual environment by running `deactivate`
```